### ISSUE TITLE
[Accessibility][Download][Sample CSS] Image information does not have sufficient contrast

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1724]
**Steps to Reproduce**
1. Use a color contrast tool to test the contrast of text and background for image information elements.

**Current Behavior**
Text for image information (filename, file size, and file type) is #737373 on a #ebebeb background, for a color contrast ratio of 3.98:1, which fails to meet the minimum contrast ratio of 4.5:1

**Expected behavior/code**
Text less than 18 points (24 pixels) meets a minimum contrast ratio of 4.5:1.

**Environment**
Chrome 91, Windows 10
AEM Core Components v. 2.17  

**Affected Users**
Users with limited vision; without perception of color

**WCAG SC**
1.4.3 Contrast (Minimum)

**Code snippet**
```
<dl class="cmp-download__properties">...</dl>
```

**Screenshot**
![image filename, file size, and file type are highlighted to indicate inadequate color contrast of gray text on gray background.](https://user-images.githubusercontent.com/86849351/128231411-f39bcc8e-d643-4b13-a2f8-283487f16bb5.png)

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES
